server/model/tests.go

```diff
@@ -23,12 +23,12 @@ type (
 
 	Output struct {
 		Selector SpanQuery
-		Value    string
+		Value    string `expr_enabled:"true"`
```

Comment:
Added those tags so we can choose which fields can contain expressions/statements. The idea is to use the same tags to inject resolved values into the test object as well. So we don't have to worry when adding new fields to the test/transaction struct.